examples/tutorials/core_api_mnist/const.yaml

```diff
@@ -0,0 +1,11 @@
+name: coreapi_mnist_tutorial
```

Comment:
`A bare-bones experiment configuration file to run the model_def.py script on a Determined cluster.`